components/automate-cli/cmd/chef-automate/iam.go

```diff
@@ -100,15 +100,15 @@ func newIAMUpgradeToV2Cmd() *cobra.Command {
 		"skip-policy-migration",
 		false,
 		"Do not migrate policies from IAM v1.")
+
+	// this flag is deprecated and does nothing but we don't wanna error on it
```

Comment:
should we add this flag as a cleanup task in https://github.com/chef/automate/issues/1451?

Reply:
Will add after this merges so I can have an exact line number!

Reply:
Actually, I think we are covered already, since we are planning on removing the [upgrade command completely](https://github.com/chef/automate/issues/1879).